backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -37,14 +37,16 @@
    private final AmazonS3Service amazonS3Service;
 
    private final String bucketName;
+   private final String defaultProfileURL;
```

Comment:
```
@Value("${aws.profile.pictures.bucket}")
private final String defaultProfileURL;
```

Reply:
this relates to the previous comment, will make the adjustment